src/Cake.Common/Tools/Chocolatey/ApiKey/ChocolateyApiKeySettings.cs

```diff
@@ -55,6 +55,6 @@ public sealed class ChocolateyApiKeySettings : ToolSettings
         /// Gets or sets a value indicating whether to run in allow unofficial mode.
         /// </summary>
         /// <value>The allow unofficial flag</value>
-        public bool AllowUnoffical { get; set; }
+        public bool AllowUnofficial { get; set; }
```

Comment:
This is a breaking change as it's an public API, though new so perhaps now's the time. Don't think many have taken an dependency on this yet, but for those that have their code will break.